curriculum/challenges/english/08-coding-interview-prep/rosetta-code/24-game.english.md

```diff
@@ -6,22 +6,22 @@ challengeType: 5
 
 ## Description
 <section id='description'>
```

Comment:
Since everything in the descriptions section reads like instructions, could you move all of the text from the description to the instructions section? Then we can put some other text from the Rosetta Code page into the description. Here's what I'm thinking:

```md
## Description
<section id='description'>
The <a href="http://en.wikipedia.org/wiki/24_Game" target="_blank">24 Game</a> tests one's mental arithmetic.
</section>

## Instructions
<section id='instructions'>
Implement a function that takes a string of four digits as its argument, with each digit from 1 to 9 (inclusive) with repetitions allowed, and returns an arithmetic expression that evaluates to the number 24. If no such solution exists, return "no solution exists".
  
<h4><strong>Rules:</strong></h4>
<ul>
  <li> Only the following operators/functions are allowed: multiplication, division, addition, subtraction. </li>
  <li> Division should use floating point or rational arithmetic, etc, to preserve remainders. </li>
  <li> Forming multiple digit numbers from the supplied digits is disallowed. (So an answer of 12+12 when given 1, 2, 2, and 1 is wrong). </li>
  <li> The order of the digits when given does not have to be preserved. </li>
</ul>

| Example input | Example output |
| --- | --- |
| <code>solve24("4878");</code> | <code>(7-8/8)*4</code> |
| <code>solve24("1234");</code> | <code>3*1*4*2</code> |
| <code>solve24("6789");</code> | <code>(6*8)/(9-7)</code> |
| <code>solve24("1127");</code> | <code>(1+7)*(2+1)</code> |
</section>
```

What do you think? Does that look alright to you?

Reply:
Looks better, implementing this